.ruby-gemset

```diff
@@ -0,0 +1 @@
+tutorialreport_xyz
```

Comment:
what does this file do?

Reply:
It's a gemset that was made so that whenever you change into this project from another, this is the gemset that will be used.